CHANGELOG.md

```diff
@@ -0,0 +1,38 @@
+# Changelog
+
+[npm][1]
+
+[1]: https://www.npmjs.com/package/@google-cloud/vision
+
+## v0.21.0
+
+### Implementation Changes
+
+- Re-generate library using /synth.py (#105)
+- Re-generate library using /synth.py (#100)
+- Re-generate library using /synth.py (#91)
+- chore: Re-generate library using /synth.py (#88)
+
+### Dependencies
+
+- fix: update and cleanup dependencies (#102)
+- fix(deps): update dependency yargs to v12 (#103)
+- chore(package): update eslint to version 5.0.1 (#98)
+- chore(package): update eslint to version 5.0.0 (#90)
+- chore: update sample lockfiles (#95)
+- repo: setup greenkeeper.json to update dependencies in samples/package.json (#94)
+- refactor: drop dependency on extend (#85)
+- fix: move async to dev dependencies (#83)
+- fix: drop support for node 4.x and 9.x (#82)
+- chore: update many dependencies (#81)
+- fix: switch from node-uuid to uuid (#84)
+
+### Documentation
+
+- fix: update linking for samples (#92)
```

Comment:
This wasn't really a docs change.  How did you decide it was?

Reply:
this is manual work to divide the categories. It is basically gut feel on the message provided.

Reply:
I have *no* context in this repo, so it's very likely I categorized some things incorrectly. Again, that's the whole point of a human review in this process.

Reply:
Got it.  I didn't know how much of this was automated vs hooman.

Reply:
Releasetool opens your editor with the template categories and an ordered list of changes, it's hooman to categorize. I am not a good hooman to categorize stuff in this repo.